app/assets/stylesheets/components/_inputs.scss

```diff
@@ -0,0 +1,27 @@
+input[type=text] {
```

Comment:
This will just target `type='text'` inputs.  You probably also want to target other similar types of inputs such as `search`, `password`, `email`, `url`, `tel` and maybe even `number`.

You can find a list of all of them here:  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Some of the new HTML5 inputs also have `:valid` and `:invalid` pseudo states that you could style along with the states you've already defined (success, error, etc...)